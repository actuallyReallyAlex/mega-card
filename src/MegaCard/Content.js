import React from 'react'
import { Box, Heading, Text } from 'grommet'
import { colors, propTypes } from '../constants'

const Content = ({ data, title, titleColor }) => {
  return (
    <Box align="start" margin={{ right: 'medium' }}>
      <Text size="16px" style={{ color: colors[titleColor] }} weight="bold">
        {title}
      </Text>
      <Heading
        color="dark-2"
        level="5"
        margin={{ bottom: 'none', top: 'xsmall' }}
        style={{ fontSize: '1.25rem' }}
      >
        {data}
      </Heading>
    </Box>
  )
}

Content.propTypes = {
  data: propTypes.data,
  title: propTypes.title,
  titleColor: propTypes.titleColor
}

export default Content
