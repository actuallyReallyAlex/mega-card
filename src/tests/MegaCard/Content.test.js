import React from 'react'
import Content from '../../MegaCard/Content'
import { shallow } from 'enzyme'

const defaultProps = {
  data: 'Test Data',
  title: 'Title'
}

describe('<Content />', () => {
  afterEach(() => {
    // defaultProps.handler.mockClear()
  })

  test('Should render the <Content /> component.', () => {
    const component = shallow(<Content {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
