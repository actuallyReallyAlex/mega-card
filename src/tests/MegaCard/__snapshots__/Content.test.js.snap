// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> Should render the <Content /> component. 1`] = `ShallowWrapper {}`;
